cmd/terramate/cli/telemetry/telemetry.go

```diff
@@ -0,0 +1,217 @@
+// Copyright 2024 Terramate GmbH
+// SPDX-License-Identifier: MPL-2.0
+
+package telemetry
+
+import (
+	"bytes"
+	"context"
+	"crypto/rand"
+	"encoding/hex"
+	"encoding/json"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/rs/zerolog/log"
+	"github.com/terramate-io/terramate"
+)
+
+type PlatformType int
```

Comment:
just noticed that I broke the linter a while ago... Those public types need documentation.

I will fix the linting in a separate PR.

Reply:
Done in [a1dbd15](https://github.com/terramate-io/terramate/pull/1981/commits/a1dbd15b5b11f2698b30434d22173b06d9a5a337)